src/components/common/UploadView/index.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+
+import { useRef } from 'react';
+
+import Plus from 'public/assets/icon/plus.svg';
+
+export default function UploadView() {
+  const imageUploadButtonRef = useRef<HTMLInputElement | null>(null);
+
+  const handleClickUploadButton = () => {
+    imageUploadButtonRef?.current?.click();
+  };
+
+  return (
+    <div className="rounded-[24px] text-center align-middle h-[160px] flex flex-col justify-center">
+      <div>
+        <p className="body-16-bold">사진을 추가해주세요.</p>
+        <p className="py-[8px] body-14-regular">
+          *기록당 최대 1개 업로드가 가능해요.
+        </p>
+        <button onClick={handleClickUploadButton}>
+          <Plus />
+        </button>
+        <input className="hidden" type="file" ref={imageUploadButtonRef} />
+      </div>
+    </div>
```

Comment:
먼가 저랑 css 작성 스타일이 많이 다른 것 같군여..
저라면 이런느낌으로 했을것같다~ 정도로 봐주시면 좋을 것 같아요!

```suggestion
    <div className="flex flex-col justify-center items-center w-full h-[160px] rounded-[24px] text-center gap-[8px]">
      <p className="body-16-bold">사진을 추가해주세요.</p>
      <p className="body-14-regular">*기록당 최대 1개 업로드가 가능해요.</p>
      <button onClick={handleClickUploadButton}>
        <Plus />
      </button>
      <input className="hidden" type="file" ref={imageUploadButtonRef} />
    </div>
```

Reply:
저도 굳이 `div`를 한 번 더 감싸야하나.. 라는 고민이 있었는데! 감사합니다.